Dockerfile

```diff
@@ -6,11 +6,13 @@ RUN apt-get update \
         make \
         g++ \
         libssl-dev \
+        nano \
+        tmux \
     && rm -r /var/lib/apt/lists/*
 
-RUN git clone https://github.com/snwagh/falcon-public.git Falcon
-
-RUN cd Falcon \
-    && make all -j$(nproc)
+RUN git clone https://github.com/trhieung/falcon-public Falcon
```

Comment:
If you're pushing these changes upstream, can you edit the repo address to link to `https://github.com/snwagh/falcon-public.git`.